modules/microshift-storage-classes.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// microshift/volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="microshift-storage-classes_{context}"]
+== Storage classes
+
+Storage classes provide the workload layer interface for selecting a device class. The following storage class parameters are supported in {product-title}:
+
+* The `csi.storage.k8s.io/fstype` parameter selects the filesystem types. Both `xfs` or `ext4` filesystem types are supported.
+* The `topolvm.io/device-class` parameter is the name of the device class. If a device class is not provided, the default device class is assumed.
+
+Multiple storage classes can refer to the same device class. You can provide varying sets of parameters for the same backing device class, such as `xfs` and `ext4` variants.
+
+.{product-title} defines the following as the default storage class:
+
```

Comment:
It might help to orient customers by labeling the example with a file name, such as `.Example persistent volume claim (PVC)` or `.Example storage class resource`